misc/readmes/profile_developers_guide.txt

```diff
@@ -86,6 +93,14 @@ the routing engine:
    - turnInstructionCatchingRange  default 40m
    - turnInstructionRoundabouts     default=true=generate explicit roundabout hints
 
+  - variables to modify BRouter behaviour
+  
+   - processUnusedTags  ( default is false )
+         If an OSM tag is unused within the profile, BRouter totally ignores the tag existence
+         what increases BRouter speed. As a side effect, the tag is not even listed
+         in the route segment table nor the table exported as CSV. Setting it to true/1,
+         Brouter-web Data page listed all tags present in the RD5 file.
```

Comment:
"Brouter-web Data page will list all tags present in the RD5 file."

Reply:
Suggestions applied and pushed.